automod/event.go

```diff
@@ -201,7 +242,38 @@ func (e *RepoEvent) PersistAccountActions(ctx context.Context) error {
 		needsPurge = true
 	}
 	for _, mr := range newReports {
-		_, err := comatproto.ModerationCreateReport(ctx, xrpcc, &comatproto.ModerationCreateReport_Input{
+		// before creating a report, query to see if automod has already reported this account in the past week for the same reason
```

Comment:
Suggest pulling this whole block out to a function.  Looks like it should have a pretty simple `(bool, error)` return type.

Reply:
:+1: